swan-lake/learn/running-ballerina-code.md

```diff
@@ -54,139 +47,92 @@ You can compile a source file with an entry point into an executable jar.
+Consider the below `Config.toml` file.
 ```toml
-[hello.user]
-name="Ballerina"
+id = 1001
+name = "Jhone"
 ```
+ Since `Config.toml` file contains a value for key `name`, the program default
+value will be overridden by the value in the `Config.tomal` file.
 
-When running a program with config API lookups, Ballerina looks for a `ballerina.conf` file in the directory where the source files are located.
-
-If `ballerina.conf` resides in the same directory as `main.bal`, `balllerina run` can be used without any argument.
 ```bash
 $ bal run main.bal
-Hello, Ballerina !
-```
-To explicitly specify a configuration file, use the `--b7a.config.file` property. The path to the configuration file can be either an absolute or a relative path. 
-```bash
-$ bal run main.bal --b7a.config.file=path/to/conf/file/custom-config-file-name.conf
-Hello, Ballerina !
-```
-
-### Configuring Secrets as Configuration Items
-Ballerina provides support for encrypting sensitive data such as passwords and allows access to them securely through the configuration API in the code.
-
-#### Creating a Secured Value
-The `ballerina encrypt` command will encrypt parameters that can be securely sourced from your code files. For example, let's create a secure parameter named `Ballerina` with the value `12345` as the secret.
-
-```ballerina
-$ bal encrypt
-Enter value:
-Enter secret:
-Re-enter secret to verify:
-Add the following to the runtime config:
-<key>="@encrypted:{Z1CfAJwCEzmv2JNXIPnR/9AXHqOJqnDaaAQ7HsggGLQ=}"
-
-Or add to the runtime command line:
---<key>=@encrypted:{Z1CfAJwCEzmv2JNXIPnR/9AXHqOJqnDaaAQ7HsggGLQ=}
+User ID : 1001
+User Name : Jhone
+Married : true
 ```
 
-#### Using the Secured Value at Runtime
-The secured value can be placed in a config file as a value or passed on the command line. 
-
-```
-[hello.user]
-name="@encrypted:{Z1CfAJwCEzmv2JNXIPnR/9AXHqOJqnDaaAQ7HsggGLQ=}"
-```
-
-or (Enter secret `12345` when prompted.):
-
-```bash
-$ bal run main.bal --hello.user.name=@encrypted:{Z1CfAJwCEzmv2JNXIPnR/9AXHqOJqnDaaAQ7HsggGLQ=}
-ballerina: enter secret for config value decryption:
-
-Hello, Ballerina !
-```
-
-#### Decrypting the Value
-If a configuration contains an encrypted value, Ballerina looks for a `secret.txt` file in the directory where the source files are located. The `secret.txt` should contain the secret used to encrypt the value. The `secret.txt` file will be deleted after it is read.
-
-```bash
-$ echo 12345 > secret.txt
-$ bal run main.bal --b7a.config.file=ballerina.conf
-Hello, Ballerina !
-```
-
-Alternatively, you can pass the path to this `secret.txt` file as a flag via the CLI as follows:
-
-```bash
- ballerina run main.bal --b7a.config.secret=<PATH_TO_SECRET_FILE>
- ```
-
-If the `secret.txt` file is not present, then CLI prompts the user for the secret. Enter secret `12345` when prompted.
+When running a program with configurable values, Ballerina looks for a `Config.toml` file in the directory of the source files. 
```

Comment:
```suggestion
When running a program with configurable values, Ballerina looks for a `Config.toml` file in the current working directory. 
```

Reply:
IMO default case should be project. We need to chnage this sample to project.